docs/how-to-guides/add-conditional-control-to-a-flow.md

```diff
@@ -47,11 +47,11 @@ activate:
 ::::
 
 ### Further details and important notes
-1. If the node using the python tool has an input that references a node that may be bypassed, please provide a default value for this input. This helps prevent errors in the current node due to missing parameters.
+1. If the node using the python tool has an input that references a node that may be bypassed, please provide a default value for this input. This helps prevent errors in the current node due to missing parameters. If there is no default value for input, the output of the bypassed node will be set to None.
```

Comment:
We will provide None for parameters that don't have default value, so this won't cause a missing parameter error. Please update here too.

Reply:
Updated.